definitions/bulk-action/json/CaseEventToFields-shareACase.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "LiveFrom": "18/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorCreate",
+    "CaseFieldID": "ApplicantOrganisationPolicy",
+    "PageFieldDisplayOrder": 20,
+    "DisplayContext": "OPTIONAL",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "applicantRepresented=\"Yes\" OR  isAdmin=\"No\"",
```

Comment:
For us I think we don't need the fieldShowCondition.
Actually those 2 fields (i.e. applicantRepresented and isAdmin) don't exist in our config so this will probably give you an error when you try to upload to AAT in case you tried to upload and saw an error :)

Reply:
Yeah it does create an error. I wanted to have it in in the early commits as reference for myself as i wasnt sure if it was meant to be conditional or not. And yes, it fails the build at the moment. Ill remove it and see if that was the only issue